Postfix/postfix/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+email.from:
+  description: Author email address
+  name: email.from
+  short: Author email address
+  type: string
+email.to:
+  description: Recipient email address
+  name: email.to
+  short: Recipient email address
+  type: string
```

Comment:
This mapping is in conflict with the official ECS specification. It causes a lot of trouble currently.